drivers/azure/azureutil/azureutil.go

```
@@ -481,6 +479,33 @@ func (a AzureClient) DeleteVirtualMachineIfExists(ctx context.Context, resourceG
 	return nil
 }
 
+func (a AzureClient) removeManagedDisk(ctx context.Context, resourceGroup, vmID, diskName string) error {
+	log.Debug("Attempting to remove managed disk.", logutil.Fields{"vm": vmID, "diskName": diskName})
+	disksClient := a.disksClient()
+	disk, err := disksClient.Get(ctx, resourceGroup, diskName)
```

This error is not being checked